pyls/config/pylint_conf.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Palantir Technologies, Inc.
+import logging
+import os
+from pyls._utils import find_parents
+from .source import ConfigSource, _get_opt, _set_opt
+
+log = logging.getLogger(__name__)
+
+PROJECT_CONFIGS = ['.pylintrc', 'pylintrc']
+
+CONFIG_KEYS = {  # 'option': 'section key'
+    'disable': 'MESSAGES CONTROL',
+    'ignore': 'MASTER',
+    'max-line-length': 'FORMAT',
+}
+
+OPTIONS = [
+    ('disable', 'plugins.pylint.disable', list),
+    ('ignore', 'plugins.pylint.ignore', list),
+    ('max-line-length', 'plugins.pylint.maxLineLength', int),
+]
+
+
+class PylintConfig(ConfigSource):
+    """Parse pylint configurations."""
+
+    def user_config(self):
```

Comment:
Would be nice to add a docstring with some explanation. Also maybe saying what the returned object is.